azurerm/data_source_managed_disk.go

```diff
@@ -53,6 +53,16 @@ func dataSourceArmManagedDisk() *schema.Resource {
 			},
 
 			"tags": tagsSchema(),
+
+			"disk_iops_read_write": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"disk_mbps_read_write": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
```

Comment:
It would be awesome if you could add these attributes to the Data Source documentation as well.

Reply:
Thanks for your suggestion. I've added the description to md file. Thanks.